src/api/parser/src/parser.js

```diff
@@ -58,10 +58,10 @@ const processFeeds = (feeds) => {
  */
 const processAllFeeds = async () => {
   try {
-    // Get an Array of Feed objects from the wiki feed list and Redis
-    const [all, wiki] = await Promise.all([Feed.all(), getWikiFeeds()]);
```

Comment:
`Feed.all()` and `getWikiFeeds()` each returns all feeds => `Feed.all()` not needed and easier to test